components/tooltip/script.js

```diff
@@ -75,7 +89,10 @@ class Tooltip extends HTMLElement {
             left: elementSize.left + (elementSize.width / 2) - tooltipSize.width / 2
         }
 
-        switch (this.position) {
+        const scrollOffsetX = window.scrollX;
+        const scrollOffsetY = window.scrollY;

```

Comment:
We can move these both to the show() method to set them once on show instead on each iteration.

Reply:
Yes - moved.